flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html

```diff
@@ -52,12 +74,22 @@ <h2 key={error}> {error} </h2>
                                     </template>
                                     <lightning-layout-item size=12>
                                         <lightning-button
-                                                class="slds-m-top_small slds-float--right"
-                                                variant="neutral"
+                                                class="slds-m-top_small slds-p-left_small slds-float--right"
+                                                variant="brand"
                                                 type="submit"
                                                 name="save"
                                                 label="Save">
                                         </lightning-button>
+                                        <template if:true={flowDisplayCancelButton}>
+                                            <lightning-button
```

Comment:
In the docs, can you provide a sample image and explain how the button interactions work.

Reply:
Will do - I hadn't started on the documentation updates yet.